### ISSUE TITLE
Firebase

### ISSUE THREAD

**Issue by tryforjunk273824724** [WebDevSimplified/Whatsapp-Clone#8]
Does anyone have a way to store this on firebase and not your cache? This would make it a way better app.

**Comment by pkboe** [comment]
Yeah, Using Realtime storage and then store all that thing in cache to avoid code crash. just to ensure that new incoming data must not be set in Local Storage in repeats
